examples/complete/main.tf

```diff
@@ -22,13 +22,13 @@ module "subnets" {
 }
 
 module "ec2_instance" {
-  source  = "cloudposse/ec2-instance/aws"
-  version = "0.38.0"
-
+  source          = "cloudposse/ec2-instance/aws"
+  version         = "0.38.0"
+  name            = "myfirstpr"
   vpc_id          = module.vpc.vpc_id
   subnet          = module.subnets.private_subnet_ids[0]
   security_groups = [module.vpc.vpc_default_security_group_id]
-  ami             = "ami-009b28ad8707b9ee8"
+  ami             = "ami-02e136e904f3da870"
```

Comment:
Let's not change this ami... It really should be pulling the ami id from a data source as this could be any ami even a malicious one.

Reply:
That ami in the original code is an ami that needs patching so it needs to stay the same